pyrometheus/codegen/python.py

```diff
@@ -286,6 +293,69 @@ def get_mixture_internal_energy_mass(self, temperature, mass_fractions):
         emix = self.get_mass_average_property(mass_fractions, e0_rt)
         return self.gas_constant * temperature * emix
 
+    def get_species_viscosities(self, temperature):
```

Comment:
Document these.

Reply:
Added the theoretical formulation  and description in model.rst.